HW_1/task_2.py

```diff
@@ -1 +1,24 @@
-# Решили 2задачу из 1 домашнего задания
\ No newline at end of file
+# Задача 4: Петя, Катя и Сережа делают из бумаги журавликов. Вместе
+# они сделали S журавликов. Сколько журавликов сделал каждый
+# ребенок, если известно, что Петя и Сережа сделали одинаковое
+# количество журавликов, а Катя сделала в два раза больше журавликов,
+# чем Петя и Сережа вместе?
+
+# 6 -> 1 4 1
+# 24 -> 4 16 4
+# 60 -> 10 40 10
+
+a = int(input('Введите количество журавликов: '))  
+u=a%3
+
+if u != 0:
```

Comment:
Это переменная, используется один раз в условий. Почему бы явно это выражение в условий и не передать.
```suggestion
if a % 3 != 0:
```